viewing_party/main.py

```diff
@@ -0,0 +1,172 @@
+def create_movie(title, genre, rating):
+    movie_dic = {}
+    if title and genre and rating:
+        movie_dic["title"] = title
+        movie_dic["genre"] = genre
+        movie_dic["rating"] = rating
+        return movie_dic
+    else:
+        return None
```

Comment:
This is a little bit cleaner if you directly define the dictionary instead of modifying it.

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
    else:
        return None
```